app/models/user.rb

```diff
@@ -85,6 +85,15 @@ def current_subscription
     current_subscriptions.first
   end
 
+  def validate_user_subscription_plan(event)
+    if event.category == 'Mob'
+      subscription = current_subscription
+      subscription.plan.id > 2 unless subscription.nil?
```

Comment:
I get this although we are relying on the plan id sequence - fine for now, but might need something more robust in future ...

Reply:
I agree! It seems like a flaky way of doing things because it relies on my knowledge of the plan id and their corresponding levels and is not easily understood by others who haven't used pry to look at the inner workings. I'll have a think to see if I can come up with a better way.